Update simple-local-avatars version to 2.8.5

roborourke opened the pull request:
Update the plugin dependency to use the wpackagist version.

This resolves the issue that the plugin expects the `10up/wp-compat-validation-tool` to be local to the plugin code and modified to have a different namespace.

The CMS module needs to initialise the plugin from the `WP_PLUGIN_DIR` directory.

- Resolves https://github.com/humanmade/product-dev/issues/1834

roborourke committed:
Update simple-local-avatars version to 2.8.5

Patchstack is reporting 2 low priority CVEs for <= 2.7.1 and <= 2.8.4

mikelittle commented:
The multiple change logs for this don't look too bad. But for such a large leap, we need to do some testing.
I'll follow up on that.

roborourke commented:
Yeah sorry, you should remove the back port labels and I’ll leave it to your discretion

mikelittle reviewed (changes requested):
I can't get this to install in product dev.
Since 2.8.0 it now requires `10up/wp-compat-validation-tool` at `dev-trunk` which is not published on Packagist.

If I manually add the git repo as a source (not gonna happen for production), it still won't install as that needs an incompatible version of "composer/installers"

Have you got this to work on a project?

mikelittle committed:
Merge branch 'master' into sla-update-2.8.5

mikelittle committed:
Update the plugin dependency to use the wpackagist version.

This resolves the issue that the plugin expects the `10up/wp-compat-validation-tool` to be local to the plugin code and modified to have a different namespace.

the CMS module needs to initialise the plugin from the `WP_PLUGIN_DIR` directory.

mikelittle committed:
Restore the plugin dependency to the packagist.org version.
Add initialisation code to ~move~copy the code to tghhe corerect diectory and search and replace the namespace in all PHP files.

However, **this won't work** on the AWS stack because of the read only file system.
